orc8r/cloud/go/services/configurator/storage/sql_test.go

```diff
@@ -539,6 +540,80 @@ func TestSqlConfiguratorStorage_LoadEntities(t *testing.T) {
 			EntitiesNotFound: []*storage.EntityID{{Type: "hello", Key: "world"}},
 		},
 	}
+	// Load  of 2 entities
+	loadPageRegex := regexp.MustCompile("SELECT ent\\.network_id, ent\\.pk, ent\\.\"key\", ent\\.type, ent\\.physical_id, ent\\.version, ent\\.graph_id, ent\\.name, ent\\.description, ent\\.config " +
+		"FROM cfg_entities AS ent " +
+		"WHERE \\(ent\\.network_id \\= \\$1 AND ent\\.type \\= \\$2 AND ent\\.\"key\" \\> \\$3\\) " +
+		"ORDER BY ent\\.\"key\" ASC " +
+		"LIMIT 2")
```

Comment:
For multi-line arg to function, the opening and closing parens should be on their own line. E.g.

```
MustCompile(
  "SELECT ..." +
  "FROM ..." +
  ...
  "LIMIT ..."
)